pallets/loans/src/functions.rs

```diff
@@ -72,14 +72,15 @@ impl<T: Config> Pallet<T> {
 		pool_id: PoolIdOf<T>,
 		loan_id: T::LoanId,
 		f: F,
-	) -> Result<R, DispatchError>
+	) -> Result<(u32, R), DispatchError>
```

Comment:
The point of this is to propagate the number of active loans up and to the top-level extrinsic fn for weight calculations (coming once we have benchmarking updated)